docs/core/diagnostics/distributed-tracing.md

```diff
@@ -0,0 +1,229 @@
+---
+title: Distributed tracing - .NET
+description: An introduction to .NET distributed tracing.
+ms.date: 02/02/2021
+---
+# .NET Distributed Tracing
+
+Distributed tracing is the way to publish and observe the tracing data in a distributed system.
+.NET Framework and .NET Core has been supporting tracing through the <xref:System.Diagnostics> APIs.
+
+- <xref:System.Diagnostics.Activity?displayProperty=nameWithType> class which allows storing and accessing diagnostics context and consuming it with logging system.
+- <xref:System.Diagnostics.DiagnosticSource?displayProperty=nameWithType> that allows code to be instrumented for production-time logging of rich data payloads for consumption within the process that was instrumented.
+
+Here is an example shows how to publish tracing data from the HTTP incoming requests:
```

Comment:
I'd recommend we not lead with this example because it is refering to the older APIs and it is unlikely that most developers are going to be writing their own protocol servers. Instead we could jump right down to the .NET 5.0 stuff